### PR TITLE
Django 3.2 compatibility

### PR DESCRIPTION
**Pull request by wamberg**
All tests pass for Django==3.2.

Changes include:
- Using `caches.create_connection` to access cache
- Cast PosixPath to str in `sync_templates`

[#116]

### CODE REVIEW

**Review by nschlemm — APPROVED** [review]
Thanks, @wamberg ! Looks good to me for 3.2, although *ideally* tox.ini should reflect the currently [supported versions](https://www.djangoproject.com/download/#supported-versions) IMHO

**Comment by wamberg** [comment]
> Looks good to me for 3.2, although _ideally_ tox.ini should reflect the currently [supported versions](https://www.djangoproject.com/download/#supported-versions) IMHO

Thanks for the review, @nschlemm. Give me a couple days and I'll update the `tox.ini` file in this PR.

**Comment by wamberg** [comment]
Hi @nschlemm, looks like you modified the `tox.ini` for supported Django versions in bcf7dcf. Do you want me to incorporate those changes into this PR? Alternatively, you can merge this PR and follow up with your changes in a subsequent PR. I'm happy with whichever way you prefer.

**Comment by nschlemm** [comment]
Hi @wamberg that was an accidental push of local changes - added a cleaner PR https://github.com/wamberg/django-dbtemplates/pull/1 for your PR instead ;)

**Comment by wamberg** [comment]
Alright, @nschlemm, I merged your PR into this one. Thanks for that! I believe this PR is ready to merge at your leisure.

**Comment by fjbardelli** [comment]
it is possible to do the merge, i need migrate to Django 3.2

**Comment by decaluweconsulting** [comment]
Hey guys, when will this be merged? Trying to update to Django 3.2.9 but incompatibility on dbtemplates package.

**Comment by ianfitzpatrick** [comment]
Would love to see this merged in as well! Django 3.2 is a LTS version, and previous LTS is out of support. But am blocked by this add on until this is merged in.

Thanks much, and appreciate the work that went into making this, has been super helpful for us 🙏

**Comment by smaccona** [comment]
We are also blocked on upgrading to Django 3.2, and as previously noted 3.2 is an LTS version.

Let me know if I can help get this PR merged and a release pushed to PyPi. Thanks!

**Comment by selwin** [comment]
@nschlemm I also have a few projects that I'd like to upgrade. Let me I can do to help get this released :)

**Comment by Natureshadow** [comment]
A merge and, most importantly, a release to PyPI would be very welcome, still!

**Comment by traverseda** [comment]
No luck, huh?

**Comment by mpasternak** [comment]
@traverseda @wamberg commited, thanks. Looks like I have repo privileges, I contacted @jezdez about PyPI release.

**Pull request merged**
